fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1289,7 +1289,7 @@ Future< Standalone<RangeResultRef> > ReadYourWritesTransaction::getRange(
 	}
 
 	// special key space are only allowed to query if both begin and end are in \xff\xff, \xff\xff\xff
-	if (specialKeys.contains(begin.getKey()) && specialKeys.contains(end.getKey()))
+	if (specialKeys.contains(begin.getKey()) && end.getKey() <= specialKeys.end)
```

Comment:
Good catch, there’s a related change that needs to be done in FuzzApiCorrectness where the contract for get range is defined.

Reply:
Thanks for fixing. My bad for confusing the boundary.

Reply:
Done